libsolidity/parsing/Parser.cpp

```diff
@@ -1981,6 +1986,26 @@ pair<vector<ASTPointer<Expression>>, vector<ASTPointer<ASTString>>> Parser::pars
 	return ret;
 }
 
+optional<string> Parser::findLicenseString()
+{
+	// We circumvent the scanner here, because it skips non-docstring comments.
+	static regex const licenseRegex("SPDX-License-Identifier:\\s+([^\\n\\r]+)");
```

Comment:
This might be the first-ever compliant contract: https://github.com/axic/eth2-deposit-contract/commit/3aaf6f06ad970b62c305f162edc848725f3d3d9a

(Pushed yesterday)

Reply:
I wonder what prompted me to start this...